service/state/state.go

```diff
@@ -16,3 +16,9 @@ type TxResponse = sdk.TxResponse
 
 // Address is an alias to the Address type from Cosmos-SDK.
 type Address = sdk.Address
+
+// AccAddress is an alias to the AccAddress type from Cosmos-SDK.
+type AccAddress = sdk.AccAddress
```

Comment:
What is the diff between those Accs and why do we need both?

Reply:
Should we try to unify those?

Reply:
cc @liamsi

Reply:
we can use `AccAddress` instead of `Address` but not vice versa bc `NewMsgSend` requires `AccAddress`.
The difference is that `Address` is an interface and `AccAddress` is an alias of `[]byte` that implements `Address` interface

Reply:
Then you can do type assertion to solve the problem without type pollution.